conbench/entities/commit.py

```diff
@@ -546,7 +547,7 @@ def _parse_commit(commit):
             "parent": commit["parents"][0]["sha"] if commit["parents"] else None,
             # Note(JP): this might need attention with respect to time zones.
             # Also see https://github.com/PyGithub/PyGithub/issues/512#issuecomment-1362654366
```

Comment:
I think you can remove this comment now 🙂

Reply:
Updating the comment in an upcoming PR

Reply:
done here: https://github.com/conbench/conbench/pull/612